Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -321,15 +315,8 @@ - (void)setSecureTextEntry:(BOOL)secureTextEntry
 {
   NSMutableDictionary<NSAttributedStringKey, id> *textAttributes = [_defaultTextAttributes mutableCopy] ?: [NSMutableDictionary new];
 
-  if (self.placeholderColor) {
-    [textAttributes setValue:self.placeholderColor forKey:NSForegroundColorAttributeName];
-  } else {
-#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
-    [textAttributes setValue:defaultPlaceholderTextColor() forKey:NSForegroundColorAttributeName];
-#else
-    [textAttributes removeObjectForKey:NSForegroundColorAttributeName];
-#endif // ]TODO(macOS ISS#2323203)
-  }
+  [textAttributes setValue:self.placeholderColor ?: [RCTUIColor placeholderTextColor]
```

Comment:
This is available in macOS 10.10, but on iOS its only 13 and higher.   I guess its ok: our deployment target is 13.2, but its lower in core.

Reply:
Ooh, good catch re the iOS deployment target. Taking into account what I said above, I’d then be inclined to add a `@available(iOS 13.0, *)` here too and in the `else` clause use the old removed code that only sets when non-nil and removes the key otherwise. How does that sound?

Reply:
Done 👍